shell/platform/tizen/tizen_log.cc

```diff
@@ -13,6 +13,16 @@ static pthread_t stdout_thread;
 static pthread_t stderr_thread;
 static bool is_running = false;
 
+// Handle filtering of logs
+static log_priority prio_min_ = DLOG_ERROR;
```

Comment:
Change name to more descriptive, like `min_priority_level` or at least `min_priority_`